### DISCUSSION TITLE
Is there any documentation for using this package in a Wpf application using a service host in the application

### DISCUSSION THREAD

**Discussion by tacoWanneeJama** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3660] [Q&A] [answered]
The manual is very clear how to implement this package in a basic Wpf application. 
Now I need to add a backend service and a logger service. Therefore I introduced a host to support DI, but now the Material design package resource has to be loaded in a different way. I still have not figured out how to get it loaded now.

**Comment by Keboo** [comment, marked as answer]
I do have a dotent template that does the setup for this. 

You can install it by running `dotnet new install Keboo.Dotnet.Templates`. You can then create a WPF app using `dotnet new keboo.wpf`,

Repo: https://github.com/Keboo/DotnetTemplates

**Comment by tacoWanneeJama** [comment]
Thanks a lot. 
That was the example I needed. Taking this example application as reference I could fix it.
